tensorflow/lite/arena_planner.cc

```diff
@@ -135,7 +135,7 @@ TfLiteStatus ArenaPlanner::PlanAllocations() {
   }
 
   // Count references to node input tensors.
-  for (size_t i = 0; i < graph_info_->num_nodes(); ++i) {
+  for (int i = 0; i < static_cast<int>(graph_info_->num_nodes()); ++i) {
     const TfLiteNode& node = graph_info_->node(i);
```

Comment:
Hmm, doesn't node(i) take a size_t?

Reply:
@jdduke , thanks for the comments, i have updated the same here and other places as well.

Regards
Amit